queue/src/main/resources/org/killbill/queue/dao/QueueSqlDao.sql.stg

```diff
@@ -207,6 +207,19 @@ updateOnError(tableName) ::= <<
     ;
 >>
 
+updateCreatingOwner(tableName) ::= <<
+    update <tableName>
+    set
+      creating_owner = :newCreatingOwner
+      , processing_owner = NULL
+      , processing_available_date = :now
```

Comment:
I believe `processing_available_date` should be `NULL` instead, as it is used as a threshold at which nodes can steal entries from other nodes in a non-STICKY mode (cc @sbrossie for confirmation).

Reply:
@pierre @wwjbatista yes, `processing_available_date` should be set to null.

Also, couple of things:
1. We should order things to start from older entries (valid if the `max` is less than total number)
2. It would be good to keep track of which entries where moved and this goes back to @pierre 's comment about logging.  At the very least, the update will return the number of rows updated, we could log that.

@pierre @wwjbatista For for thoughts -- not yet actionable:
Should we extend our schema to add an `updated_date` and also a `prev_creating_owner`.  The `prev_creating_owner` would be mostly interesting to understand what happens. My worry is that we are trying to add complexity around an error case, and so when this happens, verifying things worked as expected might be good.